copy/single.go

```diff
@@ -608,13 +629,22 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, to
 	// This MIME type → compression mapping belongs in manifest-specific code in our manifest
```

Comment:
The part of the comment above this line should stay in `copyLayer`; it is not relevant to the “image already exists” case.